src/Assignments/ProblemDetails.tsx

```diff
@@ -5,6 +5,7 @@ import moment from 'moment';
 import { OverlayTrigger, Tooltip, Badge } from 'react-bootstrap';
```

Comment:
(This comment isn't actually for this line but rather the file)
This diff is massive, I think it's just things getting moved around but I would like to have a quick call to understand the changes here

It looks good to me thought